bin/GameIndex.dbf

```diff
@@ -743,7 +743,7 @@ Name   = Namco x Capcom
 Region = NTSC-Unk
 ---------------------------------------------
 Serial = SCAJ-20131
-Name   = namCollection - Namco 50th Anniversary
+Name   = NamcoCollection - Namco 50th Anniversary
```

Comment:
No, it's literally called namCollection, look at the box https://s.pacn.ws/1500/PA.28798.017.jpg?o2utd0